### ISSUE TITLE
Add VAT field to donation/subscription forms

### ISSUE THREAD

**Issue by ctheune** [rtfd/readthedocs.org#1985]
Hi,

I booked a Gold membership but only received a very meager receipt from Stripe which our accounting isn't too happy with. We can live with this one receipt but I had to cancel the membership as we don't want to have this in our books on a regular basis.

The biggest part in German receipts are:
- date (is fine on stripe)
- unique id (is fine on stripe)
- a good address who the money goes to, basically something that a judge could send stuff to (this is missing)
- taxes (none on stripe, not a problem from my side just means we won't get any VAT back or so)

So basically the address is missing.

Cheers,
Christian

**Comment by agjohnson** [comment]
Thanks for the heads up! We'll have to figure out the best way to implement this -- an optional address field wouldn't add much too this form. Whether or not Stripe emits a receipt with this data is another question.

**Comment by doubleo2** [comment]
> optional address field

@agjohnson, @ctheune can correct me if I'm wrong, but you should not need to add any fields. The receipt should display RTDs address, not the address of the person/company who is subscribing.

**Comment by agjohnson** [comment]
Okay, finally took care of this. The change to get this on the receipts was small, but annoying enough to do with Stripe. New receipts *should* have this in the PDF version (html version doesn't appear to have our information).

VAT is another issue, I suppose we just need to collect that on signup/charges. I'll repurpose this issue to get that added.
